services/comments_service.py

```
@@ -0,0 +1,34 @@
+
+def write_comments(post_id:str,data: Comment, current_user : dict):
+    posts = get_post()
+    comments = get_comments()
+
+    if any(p["post_id"] != post_id for p in posts):
+        raise HTTPException(400, "Post ID not valid")
+    #
+    # if any(u["comment_id"] ==  for data. u in comments):
+    #     raise HTTPException(400, "이미 존재하는 코멘트 입니다.")
+
+    comment_created_at = datetime.now()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)